class.jetpack-options.php

```diff
@@ -153,13 +183,28 @@ public static function get_option( $name, $default = false ) {
 	 * @param string $name Option name
 	 * @param mixed $default (optional)
 	 *
-	 * @return mixed|void
+	 * @return mixed
 	 */
 	public static function get_option_and_ensure_autoload( $name, $default ) {
-		$value = get_option( $name );
+		// In this function the name is not adjusted by prefixing jetpack_
```

Comment:
There was another error here. In this function, $name is already prefixed with 'jetpack_' so `is_network_option()` was always returning false. I used a regexp to remove it  and check against `is_network_option()`